### PR TITLE
Search with city and street name

### PR DESCRIPTION
**Pull request by azime**
* Query example : "201 avenue Jean Jaures Lievin"
> without correction :  "201 avenue Jean Jaurès Sentinelle"
> With correction : "avenue Jean Jaures Liévin" in position 10
* Query example : "201 avenue Jean Jaures Liévin"
> With correction : "avenue Jean Jaures Liévin" in position 1

* "geocoder-tester" on "iledefrance" data: 16 more tests passed

Jira: https://jira.canaltp.fr/browse/HOR-142

### CODE REVIEW

File: libs/bragi/src/query.rs [hunk 1]
```diff
@@ -113,11 +113,26 @@ fn build_query(q: &str,
...
-    let mut should_query = vec![boost_addr,
-                                boost_admin,
-                                boost_stop,
-                                boost_main_match_query,
-                                boost_zipcode_match_query];
+    let mut should_query =
+        vec![boost_addr,
+             boost_admin,
+             boost_stop,
+             rs_q::build_bool()
```
Comment (antoine-de) [inline libs/bragi/src/query.rs]: I think we should add an OR between the field non existence  and the boost

File: libs/bragi/src/query.rs [hunk 2]
```diff
@@ -142,6 +141,16 @@ fn build_query(q: &str,
                 .build())
             .build();
         should_query.push(boost_on_weight);
+
+        should_query.push(
+            rs_q::build_match("administrative_regions.label".to_string(), q.to_string())
+            .with_boost(50)
```
Comment (pbougue) [inline libs/bragi/src/query.rs]: maybe we should put that value in a variable to change both if needed?

File: libs/bragi/src/query.rs [hunk 3]
```diff
@@ -92,6 +92,7 @@ fn build_query(q: &str,
                -> rs_es::query::Query {
 
     use rs_es::query::functions::Function;
+    let boost_ad_label = 50;
```
Comment (antoine-de) [inline libs/bragi/src/query.rs]: isn't it a bit low ? did you try several values ? (and did you rerun geocoder tester after @TeXitoi changes ?)

Comment (pbougue) [inline libs/bragi/src/query.rs]: minor: i find `boost_admin_label` more readable

Comment (azime) [inline libs/bragi/src/query.rs]: @antoine-de "16 more tests passed" after @TeXitoi changes.
I modified  boost_admin_label = 100 = boost_zipecode, now "17 more tests passed"

**Comment by TeXitoi** [comment]
Don't work correctly :confused:

**Pull request closed without merging**